include/devicetree/dma.h

```diff
@@ -190,27 +190,25 @@ extern "C" {
  *     DT_DMAS_CELL_BY_NAME(DT_NODELABEL(n), config, rx) // 0x404
  *
  * @param node_id node identifier
- * @param cell_name binding's cell name within the specifier referenced
- * as "name"
  * @param name lowercase-and-underscores dma channel specifier name
+ * @param cell binding's cell name within the specifier referenced as "name"
  * @return the value of "cell" inside the dma channel specifier
  * @see DT_PHA_PHANDLE_IDX()
  */
 
-#define DT_DMAS_CELL_BY_NAME(node_id, cell_name, name) \
-	DT_PHA_BY_NAME(node_id, dmas, cell_name, name)
+#define DT_DMAS_CELL_BY_NAME(node_id, name, cell) \
```

Comment:
Are you sure about this? The above examples have the second arguments 'channel' and 'config'. Those are in fact the names of cells in the binding. So either the example needs an update or this hunk isn't correct, I think.

Reply:
The comment needed updating, if you look at the test you'll see it matches all other _BY_NAME macros.

Reply:
(commented updated)